src/main/java/org/stellar/sdk/Asset.java

```diff
@@ -32,6 +32,13 @@ public static Asset create(String type, String code, String issuer) {
     }
   }
 
+  public static Asset create(ChangeTrustAsset.Wrapper wrapped) {
```

Comment:
(wrapped)`

Reply:
Yeah, my thinking was that the conversions would be more symmetric this way. e.g. `ChangeTrustAsset.create(native)`, or `Asset.create(changeTrustAsset)`